test/machine_test.go

```diff
@@ -122,14 +122,14 @@ func TestPower(t *testing.T) {
 	}
 }
 
-func runClient(in testResource, action v1.PowerRequest_Action) (*v1.StatusResponse, error) {
+func runClient(in testResource, action v1.PowerRequest_Action, s Server) (*v1.StatusResponse, error) {
 	var opts []grpc.DialOption
 	ctx := context.Background()
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
 	opts = append(opts, grpc.WithInsecure())
-	conn, err := grpc.Dial("localhost:"+port, opts...)
+	conn, err := grpc.Dial(s.URL+":"+s.Port, opts...)
```

Comment:
this is only updated for the client and not the server right?

Reply:
yeah, only client here.